aip/client-libraries/4231.md

```diff
@@ -219,6 +219,13 @@ to the following backwards-compatibility expectations:
   libraries.
 - The addition of a `google.api.resource_reference` annotation on an existing
   field **must** be a backwards-compatible change.
+- Changing a `google.api.resource_reference` from `child_type` to `type`
+  **must** be a backwards-compatible change when the `child_type` referenced
+  is single-parented and the newly referenced `type` is that exact parent.
```

Comment:
I wonder if we could provide examples regarding what `single-parented` means, or links to other AIPs if it is already explained?  They may seem straightforward to people that are familiar with the concept of resources, but could take a while for people to figure it out by themselves if they are not.

Reply:
Fair point, the distinction is actually described earlier in this AIP under the heading `Multi-pattern resources`. Would you still like to see some reference here?

Reply:
I've updated the language to be a little more specific regarding the parts of the resource definition that are pertinent to evaluation, as well as references to the relevant sections for more examples.